meta-resin-common/recipes-containers/mkfs-hostapp-native/mkfs-hostapp-native.bb

```diff
@@ -0,0 +1,35 @@
+SUMMARY = "Resin hostapp rootfs creation tool"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://Dockerfile \
+    file://create \
+    file://mkfs.hostapp-ext4 \
+    "
+
+inherit native
```

Comment:
The preferred yocto way of defining a native recipe is using:
BBCLASSEXTEND = "native"

Reply:
done